cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -342,6 +357,22 @@ def delete(self):
             {LOWEST: 'The selected Catalog Item was deleted',
              '5.11': 'The catalog item "{}" has been successfully deleted'.format(self.name)}))
 
+    def copy(self, name=None):
+        view = navigate_to(self, 'Copy')
+
+        # there is default name like `Copy of *`
+        if name:
+            view.name.fill(name)
+
+        copied_name = view.name.value
```

Comment:
Q: Why not just use `name` here?

Reply:
so `copy` method of catalog item just asked `name`; means it copied same catalog item with new name. Catalog item name is optional else default it will take like `Copy of *`. so I'm just reading name from textbox here.

Reply:
Ah, duh! makes sense, thanks @digitronik :+1: